### ISSUE TITLE
[cape] "cape not found" 

### ISSUE THREAD

**Issue by misredletteR** [sp614x/optifine#5152]
## Description of Issue
on December 15, i had gotten myself an optifine cape. it was working and showing up just fine up until December 22. i literally have no idea how it happened, or what i could have possibly done for it to end up like that, it just suddenly stopped working.

i can't change the cape design, deactivate it, or even move it to another account. all i get whenever i try is this
![image](https://user-images.githubusercontent.com/76676412/103192737-9eb2b580-488e-11eb-8e47-74f6f3690b25.png)

and the thing is, i have a spare cape i tried to put on that same username, but putting that down only gets me this
![image](https://user-images.githubusercontent.com/76676412/103192812-e8030500-488e-11eb-94ec-fa20ca5a09a1.png)

attempting to transfer the current cape to a friend only gets me this
![image](https://user-images.githubusercontent.com/76676412/103192866-2b5d7380-488f-11eb-8dc4-6b24c8f5a741.png)

my settings have it so capes for myself and everyone are visible, so i can see everyone elses' capes just fine. 

weirdly enough, saving changes here displays this
![image](https://user-images.githubusercontent.com/76676412/103192983-9ad36300-488f-11eb-9384-3fd88f2d8b50.png)
![image](https://user-images.githubusercontent.com/76676412/103192985-9f981700-488f-11eb-85d6-1594aca3e34c.png)

but changing the cape in any way once again brings up the "cape not found" error.

so in conclusion... i am kind of stuck :(

## OptiFine Version
latest (1.16.4)

## Installation Method
standalone installer. i have not dragged it into the mods folder

## Other Installed Mods
![image](https://user-images.githubusercontent.com/76676412/103192541-e71da380-488d-11eb-997b-a146e688221b.png)

## F3 Debug Screenshot
![image](https://user-images.githubusercontent.com/76676412/103193312-f2260300-4890-11eb-9474-08a80a93f8b4.png)
(singleplayer world. and as you can see, cape nonexistent)

(...and something _somewhat_ irrelevant: i was able to get the santa hat to disappear and reappear, and even so the cape still does Not show up)

**Comment by sp614x** [comment]
The cape is fixed now.
The last design was: http://www.needcoolshoes.com/banner?=faeKjppvpibtpEpfpF

**Comment by 95Number** [comment]
Hi @sp614x  i can't change the cape design, deactivate it so  I have the same problem , could you move my cape for me? here is the informations
![ii](https://user-images.githubusercontent.com/109309161/179060262-08547d25-ea08-4f10-9ede-a74b7d0004ed.PNG)
![rrrr](https://user-images.githubusercontent.com/109309161/179060268-3fb402c4-cdcf-4d9f-812a-dc0b4987c8b8.PNG)
![Capture](https://user-images.githubusercontent.com/109309161/179060273-92b74921-77af-4136-8496-f8a8f0dfff7a.PNG)

**Comment by Redragonzz** [comment]
Hi  @sp614x, same problem for me i cant deactivate the cape or change the design either, and the cape does not show up, taking me to the same problem "Cape not found" , could you help me?
![Captura de pantalla 2022-07-14 154858](https://user-images.githubusercontent.com/109259098/179083033-a5d5c8b7-fab0-4c45-af16-e1e8c33e5aed.png)
![Captura de pantalla 2022-07-14 154934](https://user-images.githubusercontent.com/109259098/179083164-c1ab04ed-cc40-4b28-aab8-c5daba94f890.png)
![Captura de pantalla 2022-07-14 154958](https://user-images.githubusercontent.com/109259098/179083260-e8b3987a-c50f-4782-805f-4f28d3986dda.png)

**Comment by Suntrapp** [comment]
Hi @sp614x, i have the same problem as everyoe in here, cant change cape, laggy website, somtimes it says cape cahnged but nothing in game, sumtimes it says that my account does not have a cape and other times server error, i tried cleaning up my browser cache on all 3 of them and tried mozilla, chrome and opera, can you help me please.
![image_2022-07-14_171717497](https://user-images.githubusercontent.com/109314781/179087707-18a32c8b-3604-43c0-a19c-1cb66c991bb1.png)
![image_2022-07-14_171743190](https://user-images.githubusercontent.com/109314781/179087760-a7f06076-2624-4aa8-b71b-b3a1b66d0da5.png)
![image_2022-07-14_171808794](https://user-images.githubusercontent.com/109314781/179087827-bc4befea-6d76-479b-a03b-8092e68541a1.png)
![image_2022-07-14_171841918](https://user-images.githubusercontent.com/109314781/179087913-e85e5d1f-e1eb-4658-94d0-296616350739.png)

**Comment by urelybitch** [comment]
hi @sp614x i have the same problem of error: not found for urely (my nick) and i wanted your help to solve this
![Uploading 1657838144185944830729971107065.jpg…]()
![Uploading 16578382203739064041543354553332.jpg…]()
